src/lib/telemetry.js

```diff
@@ -0,0 +1,39 @@
+/*
```

Comment:
Just for consistency style-wise, can we please make a few small codestyle fixes in here?

- indents
- always use semi-colons
- consistent use of either single or double quotes

Reply:
Done.